.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@
 				},
 				{
 					name: build,
-					run: ./.github/workflows/scripts/mac/xcode-build.sh
+					run: ./.github/workflows/scripts/mac/xcode-build.sh || cat ./out.txt
```

Comment:
Should this be `> ./out.txt`?

Reply:
The Xcode build script stage isn't printing its output, so I'm writing it to out.txt, then using `cat` here to see it in the action log.